modules/nw-customize-ingress-error-pages.adoc

```diff
@@ -0,0 +1,51 @@
+// Module filename: nw-customize-ingress-error-pages.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-customize-ingress-error-pages_{context}"]
+= Customizing Ingress Controller error pages
+
+As a cluster administrator, you can use the `httpErrorCodePages` field to specify custom error code pages for the Ingress Controller. {product-title} router serves a 503 error page by default when the server is not running or when the route is incorrect or non-existent. When you specify a custom error code page, {product-title} serves either a 503 or 404 error page when the requested URL does not exist. The default HTTP 503 error page is not served.
+
+See the link:https://raw.githubusercontent.com/openshift/router/master/images/router/haproxy/conf/error-page-503.http[default 503 error page].
+
+.Procedure
+
+. Create a configmap named "my-custom-error-code-pages" in the `openshift-config` namespace:
++
+[source,terminal]
+----
+$ oc -n openshift-config create configmap my-custom-error-code-pages
+--from-file=error-page-503.http
+--from-file=error-page-404.http
+----
++
+[NOTE]
+====
+The configmap keys have two available formats, error-page-503.http and error-page-404.http. You can use one or both of the available formats.
+====
+
+. Patch the Ingress Controller to reference the `my-custom-error-code-pages` config map by name:
++
+[source,terminal]
+----
+$ oc patch -n openshift-ingress-operator ingresscontroller/default --patch '{"spec":{"httpErrorCodePages": "my-custom-error-code-pages"}}' --type=merge
+----
+
+. The `my-custom-error-code-pages` configmap in the `openshift-config` namespace copies to the `openshift-ingress-operator` namespace. You can see the copy by running the following command:
++
+[source,terminal]
+----
+$ oc get cm default-errorpages -n openshift-ingress
+NAME                       DATA   AGE
+default-errorpages         2      25s
+----
++
+The router is configured with a volume. The config map containing the custom error pages mounts on the router volume and on top of the existing error pages in the router pods.
```

Comment:
replace "The router is configured with a volume. The config map containing the custom error pages mounts on the router volume and on top of the existing error pages in the router pods." 

with the following ->


The configmap containing the custom error pages mounts on the router volume where the key of the configmap is the filename having custom http error code response.

One can check using this command 

For 503 custom http custom error code response
`oc -n openshift-ingress rsh <router pod>   cat /var/lib/haproxy/conf/error_code_pages/error-page-503.http

For 404 custom http custom error code response
`oc -n openshift-ingress rsh <router pod>   cat /var/lib/haproxy/conf/error_code_pages/error-page-404.http